src/pages/CategoriesPages.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const CategoriesPages = () => (
+  <h2>Come back later</h2>
+);
```

Comment:
- [ ] In this component, you are to have a button with text that says "Check Status" and not a h2 tag. Kindly add replace the h2 tag with a button with the text Check Status. The reason is that this button will later be clicked that displays something on the screen, but that is for a later chapter of the project 👍